src/pages/signin/SignInPage.js

```diff
@@ -81,21 +81,17 @@ class SignInPage extends Component {
         const welcomeText = this.props.translate(`welcomeText.${showPasswordForm ? 'phrase4' : 'phrase1'}`);
 
         return (
-            <>
-                <SafeAreaView style={[styles.signInPage]}>
-                    <SignInPageLayout
-                        welcomeText={welcomeText}
-                        shouldShowWelcomeText={showLoginForm || showPasswordForm || !showResendValidationLinkForm}
-                        shouldShowWelcomeScreenshot={showLoginForm}
-                    >
-                        {showLoginForm && <LoginForm />}
-
-                        {showPasswordForm && <PasswordForm />}
-
-                        {showResendValidationLinkForm && <ResendValidationForm />}
-                    </SignInPageLayout>
-                </SafeAreaView>
-            </>
+            <SafeAreaView style={[styles.signInPage]}>
+                <SignInPageLayout
+                    welcomeText={welcomeText}
+                    shouldShowWelcomeText={showLoginForm || showPasswordForm || !showResendValidationLinkForm}
+                    shouldShowWelcomeScreenshot={showLoginForm}
+                >
+                    <LoginForm visible={showLoginForm} />
+                    <PasswordForm visible={showPasswordForm} />
```

Comment:
This is change is necessary. Most password managers do not work with dynamically added fields. So When the user selected the username in LoginForm and navigate to PasswordForm that autofill will be lost and he has to refill the value from suggestions.

To make it work with them, I just hid them behind the CSS and kept them in DOM. But I think LoginForm and PasswordForm are not needed or not part of `ResendValidationForm` so these should be unmounted for `showResendValidationLinkForm`.

I can't use `{!showResendValidationLinkForm  &&  <> ...</>}` this reattach those forms and break it. If I extract them into another PureComponent then return null for `!showResendValidationLinkForm ` it will work. I need your suggestion here. Thanks.

Reply:
This will solve https://github.com/Expensify/App/issues/5956 as well.

Reply:
>When the user selected the username in LoginForm and navigate to PasswordForm that autofill will be lost and he has to refill the value from suggestions

Which autofill will be lost? If we navigate to the password form the login form value is lost?

Not following the rest of the proposal, sorry. But I think we should avoid manipulating any CSS directly.